lib/carrierwave/storage/fog.rb

```diff
@@ -487,7 +487,7 @@ def file
         def copy_options
           options = {}
           options.merge!(acl_header) if acl_header.present?
-          options['Content-Type'] ||= content_type if content_type
+          options['Content-Type'] = options[:content_type] ||= content_type if content_type
```

Comment:
```suggestion
          options[fog_provider == "Google" ? :content_type : 'Content-Type'] ||= content_type if content_type
```

How about using `:content_type` only for fog-google, since it's hard to tell if `:content_type` causes side-effects for other fog providers?

Reply:
LGTM